modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -337,6 +337,13 @@ public DDMFormInstance getFormInstance() {
 
 					_ddmFormInstance.setSettings(
 						latestApprovedDDMFormInstanceVersion.getSettings());
+
+					DDMStructureVersion structureVersion =
```

Comment:
@leticia-maciel I would rename this to `ddmStructureVersion` but I'll forwart as it is so we don't have to run CI again.